lib/mix/lib/mix/tasks/deps.tree.ex

```diff
@@ -89,6 +98,12 @@ defmodule Mix.Tasks.Deps.Tree do
 
   defp callback(formatter, deps, opts) do
     excluded = Keyword.get_values(opts, :exclude) |> Enum.map(&String.to_atom/1)
+    included = Keyword.get_values(opts, :include) |> Enum.map(&String.to_atom/1)
+
+    if included != [] and excluded != [] do
```

Comment:
I don't see an issue with allowing both. First include, then exclude.

Reply:
Sure, works for me!